modules/vg.py

```diff
@@ -18,65 +18,66 @@
-                     either a volume group creation or deletion.
-    disks:
+        choices: [create,  extend, reduce, convert, remove]
+        description: Specifies the vg operation that is to be executed on the vg
+    disk:
         required: true
         description: Physical Volumes on which the Volume Groups are to be
                      created or Volume Groups that are to be removed needs to
                      be specified here.
-    options:
-        required: false
-        description: Extra options that needs to be passed while creating the
-                     Volume Groups can be given here. Check the man page of
-                     vgcreate for more info.
-
     vgname:
-        required: true for remove action
-        description: Names of the Volume Groups that are to be removed
-
+        required: true
+        description: Names of the Volume Group
     disk_type:
         required: false
         choices: [raid10, raid6, jbod]
         description: Specifies which disk configuration is used while
-        setting up the backend. Supports RAID 10, RAID 6 and JBOD
-        configurations. Defining this will set the module to
-        assign physicalextentsize best for the performance based
-        on the number of datadisks and the stripe unit size in the
-        configuration.
-
+                     setting up the backend.
     stripe_size:
-        required: true if disk_type is provided
+        required: false
         description: Specifies the stripe unit size of each disk
-        in the architecture
-
+                     in the architecture
     diskcount:
-        required: true if disk_type is provided
-        description: Specifies the number of data disks in the
-        configuration.
+        required: false
+        description: Specifies the number of data disk in the
+                     configuration.
+    force:
+        required: false
+        description: Force  the  creation  without  any confirmation.
+    uuid:
+        required: false
+        description: Specify the uuid for the device.  Without this option,
+                     pvcreate generates a random uuid.
+    zero:
+        required: false
+        description: Whether or not the first 4 sectors of the device should
+                     be wiped.
+    metadatasize:
+        required: false
+        description: The approximate amount of space to be set aside for
+                     each metadata area.
+    dataalignment:
+        required: true
+        description: Align the start of the data to a multiple of this number.
+
 
-author: Anusha Rao, Nandaja Varma
 '''
 
 EXAMPLES = '''
-#Create Volume Groups on PVS /dev/sdb and /dev/sdc with
-#physical extension size 128k
-    - vg: action=create disks='["/dev/sdb", "/dev/sdc"]'
-          options="--physicalextentsize 128k"
+#Create Volume Groups on PVS /dev/vdb1
+    - vg: action=create disk='["/dev/vdb"]'
```

Comment:
This needs a change the syntax is different:

vg: action=create disk="{{ item }}"
with_items:
      - vgname1
      - vgname2